docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -9,59 +9,73 @@ title: Using kubeadm to Create a Cluster
 
 {% capture overview %}
 
-This quickstart shows you how to easily install a Kubernetes cluster on machines
-running Ubuntu 16.04+, CentOS 7 or HypriotOS v1.0.1+. The installation uses a
-tool called _kubeadm_ which is part of Kubernetes.  As of v1.6, kubeadm aims to
-create a secure cluster out of the box via mechanisms such as RBAC.
-
-This process works with local VMs, physical servers and/or cloud servers. It is
-simple enough that you can easily integrate its use into your own automation
-(Terraform, Chef, Puppet, etc).
-
-See the full [kubeadm reference](/docs/admin/kubeadm/) for information on all
-kubeadm command-line flags and for advice on automating kubeadm itself.
-
-kubeadm assumes you have a set of machines (virtual or real) that are up and
-running.  It is designed to be part of a large provisioning system - or just for
-easy manual provisioning.  kubeadm is a great choice where you have your own
-infrastructure (e.g. bare metal), or where you have an existing orchestration
-system (e.g. Puppet) that you have to integrate with.
-
-If you are not constrained, there are other higher-level tools built to give you
-complete clusters:
-
-* On GCE, [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/)
-  gives you one-click Kubernetes clusters.
-* On Microsoft Azure, [Azure Container Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes)
-  gives you managed Kubernetes clusters as a service.
-* On AWS, [kops](https://github.com/kubernetes/kops) makes cluster installation
-  and management easy.  kops supports building high availability clusters (a
-  feature that kubeadm is currently lacking but is building toward).
+**kubeadm** is a toolkit that help you bootstrap a best-practice Kubernetes
+cluster in an easy, reasonably secure and extensible way. It also supports
+managing [Bootstrap Tokens](#TODO) for you and upgrading/downgrading clusters.
+
+kubeadm aims to set up a minimum viable cluster that pass the
+[Kubernetes Conformance tests](#TODO), but installing other addons than
+really necessary for a functional cluster is out of scope.
+
+It by design does not install a networking solution for you, which means you
+have to install a third-party CNI-compliant networking solution yourself
+using `kubectl apply`.
+
+kubeadm expects the user to bring a machine to execute on, the type doesn't
+matter, can be a Linux laptop, virtual machine, physical/cloud server or
+Raspberry Pi. This makes kubeadm well suited to integrate with provisioning
+systems of different kinds (e.g. Terraform, Ansible, etc.).
+
+kubeadm is designed to be a good way for new users to start trying
+Kubernetes out, possibly for the first time, an way for existing users to
+test their application on and stich together a cluster easily and to be
+a building block in a larger ecosystem and/or installer tool with a larger
+scope.
```

Comment:
... users to test their applications. Kubeadm can be used also as a building block to be used in other installer or in higher level tools